src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -134,7 +134,7 @@ const WorkspaceContainer = ({ namespace, name, breadcrumbs, topBarContent, title
     ]),
     showTabBar && h(WorkspaceTabs, { namespace, name, activeTab, refresh, workspace, deletingWorkspace, setDeletingWorkspace, cloningWorkspace, setCloningWorkspace, sharingWorkspace, setSharingWorkspace }),
     div({ role: 'main', style: Style.elements.pageContentContainer },
-      // Display the context bar only if the analysis tab is visible, for now
+      // When we switch this over to all tabs, ensure other workspace tabs look the same when inside these divs
```

Comment:
This may be difficult to remember to do when we get to that stage. Is there a ticket to keep track of TODOs like this?